### PR TITLE
Fix dinosaur name and other typos

### PR DESCRIPTION
**Pull request by janekdb**

### CODE REVIEW

**Comment by janekdb** [comment]
No merge yet!

I need to

* Squash commits
* Self review

Plus I'm being lazy and running the build for the first time in this PR!

File: test/junit/scala/tools/nsc/typechecker/InferencerTest.scala [hunk 1]
```diff
@@ -145,13 +145,23 @@ class InferencerTests extends BytecodeTesting {
       assertTrue(!typer.infer.isAsSpecific(LZA, LZB))
       assertTrue(typer.infer.isAsSpecific(LZB, LZA))
 
+<<<<<<< d86afea18a973dcf3489944edd9e337997dd35ea:test/junit/scala/tools/nsc/typechecker/InferencerTest.scala
```
Comment (janekdb) [inline test/junit/scala/tools/nsc/typechecker/InferencerTest.scala]: Opps!

Comment (SethTisue) [inline test/junit/scala/tools/nsc/typechecker/InferencerTest.scala]: whoops, a merge conflict crept in here

**Review by SethTisue — CHANGES_REQUESTED** [review]
commented on one needed fix, otherwise LGTM! 🦕❤️

**Comment by SethTisue** [comment]
> No merge yet!

(GitHub's "draft" feature is good for this.)

**Review by janekdb — COMMENTED** [review]
> > No merge yet!
> 
> (GitHub's "draft" feature is good for this.)

Didn't know about "draft" feature - I will check it out!

File: test/files/run/finally.scala [hunk 2]
```diff
@@ -111,7 +111,7 @@ object Test extends App {
     try {
       m
     } catch {
-      case e: Throwable => println("COUGHT: " + e)
```
Comment (lrytz) [inline test/files/run/finally.scala:114]: 🤧

Comment (janekdb) [reply test/files/run/finally.scala:114]: Bless you!

File: test/junit/scala/tools/nsc/typechecker/InferencerTest.scala (hunk 1 above)
Comment (janekdb) [reply test/junit/scala/tools/nsc/typechecker/InferencerTest.scala]: Fixed.

**Comment by janekdb** [comment]
I've addressed all the comments and do not have any else to do on this. If this passes CI then give it a review!

File: test/junit/scala/tools/nsc/typechecker/InferencerTest.scala (hunk 1 above)
Comment (janekdb) [reply test/junit/scala/tools/nsc/typechecker/InferencerTest.scala]: Fixed.

File: src/compiler/scala/tools/nsc/backend/jvm/opt/CallGraph.scala [hunk 3]
```diff
@@ -190,7 +190,7 @@ abstract class CallGraph {
           val argInfos = computeArgInfos(callee, call, paramTps, typeAnalyzer)
 
           // A nullness analysis could be used to prevent emitting unnecessary receiver null checks
-          // when inlining non-static callistes. However, LocalOpt's nullness cleanup will also do
+          // when inlining non-static callsites. However, LocalOpt's nullness cleanup will also do
```
Comment (som-snytt) [inline src/compiler/scala/tools/nsc/backend/jvm/opt/CallGraph.scala:193]: `καλλίστη`

File: test/files/run/OrderingTest.scala [hunk 4]
```diff
@@ -20,7 +20,7 @@ object Test extends App {
     test(t2, t2);
   }
 
-  assert(Ordering[String].compare("australopithecus", "brontausaurus") < 0)
+  assert(Ordering[String].compare("australopithecus", "brontosaurus") < 0)
```
Comment (som-snytt) [inline test/files/run/OrderingTest.scala:23]: `"apatosaurus"` and `>`.

File: test/files/run/finally.scala (hunk 2 above)
Comment (som-snytt) [reply test/files/run/finally.scala:114]: Very pandemic!

**Review by SethTisue — APPROVED** [review]

**Comment by SethTisue** [comment]
thank you Janek!

**Pull request merged**